modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -8,6 +8,21 @@
 
 namespace Godot
 {
+    /// <summary>
+    /// Specifies which order Euler angle rotations should be in.
+    /// When composing, the order is the same as the letters. When decomposing,
+    /// the order is reversed (ex: YXZ decomposes Z first, then X, and Y last).
+    /// </summary>
+    public enum EulerOrder
+    {
+        XYZ,
+        XZY,
+        YXZ,
+        YZX,
+        ZXY,
+        ZYX
+    };
```

Comment:
Is this used anywhere outside Basis? If not, we may want to move it inside the Basis struct, like `Vector3.Axis`.